extras/phpcs/PHPCSDiff.php

```diff
@@ -331,10 +331,10 @@ private function filterMessages(array $messages, array $added_lines): array
     protected function getErrors(array $data, string $type = 'ERROR'): array
     {
         $errors = [];
-        if (!isset($data['files']) || count($data['files']) < 1) {
+        if (!isset($data['files']) || !is_array($data['files']) || count($data['files']) < 1) {
             return [];
         }
-        if (count($data['files']) > 1) {
+        if (is_array($data['files']) && count($data['files']) > 1) {
```

Comment:
we would never get to the condition where `$data['files']` isn't an array, thanks to the change on line 334 :eyes:

Reply:
Ah, yes. Good catch!

Reply:
```suggestion
        if (count($data['files']) > 1) {
```

so can we leave it as-is in that case?

Reply:
Yeah. I tried to leave it as-is but I messed up the rebase.